code/modules/admin/verbs/randomverbs.dm

```diff
@@ -118,6 +118,23 @@
 	message_admins("<span class='notice'>\bold GlobalNarrate: [key_name_admin(usr)] : [msg]<BR></span>", 1)
 	feedback_add_details("admin_verb","GLN") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_local_narrate()
+	set category = "Special Verbs"
+	set name = "Local Narrate"
+
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
```

Comment:
this does not actually check for any specific rights though

Reply:
holder is a admin holder thing, and all the other admin verbs make use of it, so i assume it's fine? i can make it more robust if i need to

Reply:
Holder is just the general permissions holder.
-> Everyone with some sort of permission has that (Admins, Mods, Devs, CCIA, ...)